runtime/near-test-contracts/src/lib.rs

```diff
@@ -79,6 +79,23 @@ pub fn fuzzing_contract() -> &'static [u8] {
     CONTRACT.get_or_init(|| read_contract("contract_for_fuzzing_rs.wasm")).as_slice()
 }
 
+/// Smallest (reasonable) contract possible to build from Rust code.
+pub fn smallest_rs_contract() -> &'static [u8] {
+    static CONTRACT: OnceCell<Vec<u8>> = OnceCell::new();
+    CONTRACT.get_or_init(|| read_contract("smallest_contract")).as_slice()
```

Comment:
Perhaps just use `wat_contract` here? using Rust for smallest contract feels counter-productive. 

Also, could you update comment on `wat_contract` as a drive-by? It's copy-paste error

Reply:
Yes good call, now I just removed the smallest contract crate entirely.